samples/sdl_gamecontroller/main.c

```diff
@@ -0,0 +1,67 @@
+#include <xboxrt/debug.h>
+#include <pbkit/pbkit.h>
+#include <hal/xbox.h>
+#include <assert.h>
+#include <SDL.h>
+#include <stdbool.h>
+
+int main() {
+  int ret = pb_init();
+  if (ret != 0) {
+    XSleep(2000);
+    return -1;
+  }
+
+  pb_show_debug_screen();
+
+  SDL_InitSubSystem(SDL_INIT_GAMECONTROLLER);
+
+  int gameControllerIndex = 0;
+  assert(SDL_IsGameController(gameControllerIndex));
+  SDL_GameController *gameController = SDL_GameControllerOpen(gameControllerIndex);
+  if (gameController == NULL) {
+    debugPrint("Could not open GameController %i, exiting...", gameControllerIndex);
+    SDL_QuitSubSystem(SDL_INIT_GAMECONTROLLER);
+    XSleep(2000);
+    return -1;
+  }
+
+  debugPrint("GameController name: %s\n", SDL_GameControllerName(gameController));
+  debugPrint("Is GameController  : %s\n", SDL_IsGameController(gameControllerIndex)?"Yes":"No");
```

Comment:
See comment from earlier: This should happen *before* opening it.

This can only ever be "Yes"; as `gameController` is `NULL` otherwise *and* the assertion would have failed.